Moodis/Moodis/Models/User.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Moodis.Feature.Login
+{
+    [Serializable]
+    class User
+    {
+        private String username;
+        private String password;
+
+        public User(String username, String password)
+        {
+            this.username = username;
+            this.password = password;
+        }
+
+        public String getUsername()
```

Comment:
You can change these getters to the ones in auto-implemented properties. https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/auto-implemented-properties